.github/workflows/increment-version-on-merge.yml

```diff
@@ -43,8 +43,17 @@ jobs:
       # See https://github.com/im-open/git-version-lite for more details around how to increment
       # major/minor/patch through commit messages
       - name: Increment the version
-        uses: im-open/git-version-lite@main
+        uses: ./
+        id: version
         with:
           create-ref: true
           github-token: ${{ secrets.GITHUB_TOKEN }}
           default-release-type: major
+
+      - name: Create Major Tag
+        uses: im-open/create-tags-action@v1
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          sha: ${{ steps.version.outputs.NEXT_VERSION_SHA }}
+          source-tag: ${{ steps.version.outputs.NEXT_VERSION }}
+          include-major: true
```

Comment:
If the stewards over this action do not want to use the `create-tags-action`, this tag creation can be done inline:
```suggestion
     - name: Create Major and Latest Tags
        uses: actions/github-script@v6
        with:
          github-token: ${{ secrets.GITHUB_TOKEN }}
          script: |
            const majorOnlyTag = '${{ steps.version.outputs.NEXT_MAJOR_VERSION }}';
            const sha = '${{ steps.version.outputs.NEXT_VERSION_SHA }}';
            const tagExists = async tag => {
              try {
                const { data } = await github.rest.git.getRef({
                  ...context.repo,
                  ref: `tags/${tag}`
                });
                return data ? true : false;
              }
              catch (e) {
                if (e.status === 404) return false;
                throw new Error(`Retrieving refs failed with the following error: ${e}`)
              }
            }
            const upsertRef = async tag => {
              const payload = {
                ...context.repo,
                sha
              };
              const foundTag = await tagExists(tag);
              console.debug(`Tag [${tag}] found`, foundTag);
              
              if (foundTag) {
                await github.rest.git.updateRef({
                  ...payload,
                  ref: `tags/${tag}`,
                  force: true
                });
                return;
              }  
              github.rest.git.createRef({
                ...payload,
                ref: `refs/tags/${tag}`,
              });
            };
            
            await upsertRef(majorOnlyTag);
```